chef/cookbooks/keystone/recipes/ha.rb

```diff
@@ -65,5 +65,12 @@
     only_if { CrowbarPacemakerHelper.is_cluster_founder?(node) }
   end
 
+  crowbar_pacemaker_order_if_exists "o-#{service_name}" do
```

Comment:
Sorry, I renamed to crowbar_pacemaker_order_only_existing (which makes more sense)